### PR TITLE
Milestone Two 

### PR DESCRIPTION
**Pull request by wandji20**
I have created 4 components for the project interface
- App component:
- Display component: Called in App component
- ButtonPanel component: Called in the App component
- Button component: Called in the ButtonPanel component
The buttons just receive props from the ButtonPanel component and don't do anything when clicked.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </div>
+    );
+  }
+  return (
+    <div className={isZero ? 'btn zero' : 'btn'}>
+      {value}
+    </div>
+  );
+}
+
```
Comment (uche-inyama) [inline src/components/Button.js]: - [x] Button component should accept `a button name` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png))
- [x]  Validate your props, with **prop-types**.

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Display() {
+  const input = '0';
+  return (
+    <div className="display">
+      {input}
+    </div>
+  );
+}
```
Comment (uche-inyama) [inline src/components/Display.js]: - [x] The display component should accept the **result** of the calculation as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png))
- [x] Validate your props, with **prop-types**
- [x] The **result** prop should be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi Wandji :slightly_smiling_face: 

Your project is coming along just fine. However, there are some updates you need to make.
I have left inline comments; kindly review it and implement the necessary changes.

If you need clarifications, kindly reach out on slack. @ Inyama, Uchechukwu Henry.

#### That's it, make these changes then submit another form for a code review.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Approved 👏 

Hi @wandji20, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**